lib/ActivityStreams.js

```diff
@@ -266,8 +274,12 @@ ActivityStreams.prototype = {
   },
 
   /**
-   * Removes a worker and calls callback if defined
+   * Logs telemetry event
    */
+  _logPerformance(tag, worker, data) {
```

Comment:
>  For example, I am customary using this API to place debug output and see which logs and when are generated by the application rather then the PerfMeter.

I would say that if you want to put dump statements, they should not affect how the code is written.
For your own development, you could put dump statements inside of PerfMeter's log function.

This adds unnecessary indirection which complicates the code for no obvious gain. We now have extra functions that serve no apparent function in the running of the program.

> Another reason is a potential change to the perfMeter.log() signature, for example we decide to send in yet another parameter to the perfMeter (_appURLs could be a candidate) - we only need to change one call site instead of all.

Are we doing this right now? If not, that sounds like over engineering